middleware-catalog-module/.idea/workspace.xml

```diff
@@ -1,33 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="AnalysisUIOptions">
```

Comment:
This files inside .idea directory should not be here. They are on .ignore. You need to delete them in your machine then make a git add, git commit, git push.